### PR TITLE
Fix eviction order

### PR DESCRIPTION
**Pull request by tzneal**
**1. Issue, if available:**
N/A

**2. Description of changes:**

- Update links in docs
- Fix bug in pod eviction order

**3. How was this change tested?**

Unit test only

**4. Does this change impact docs?**
- [ ] Yes, PR includes docs updates
- [ ] Yes, issue opened: *link to issue*
- [X ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by njtran — APPROVED** [review]
Thanks for the fix!

**Comment by ellistarn** [comment]
Nice to see you stop by Todd! Nice fix 💪

File: pkg/controllers/provisioning/binpacking/packer.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ type Packing struct {
 // to make better cost and availability decisions. The instance types returned are sorted by resources.
 // Pods provided are all schedulable in the same zone as tightly as possible.
 // It follows the First Fit Decreasing bin packing technique, reference-
-// https://en.wikipedia.org/wiki/Bin_packing_problem#First_Fit_Decreasing_(FFD)
+// https://en.wikipedia.org/wiki/First-fit-decreasing_bin_packing_(FFD)
```
Comment (ellistarn) [inline pkg/controllers/provisioning/binpacking/packer.go:84]: This link 404s for me.

Comment (njtran) [reply pkg/controllers/provisioning/binpacking/packer.go:84]: Works for me. Although I went here https://en.wikipedia.org/wiki/First-fit-decreasing_bin_packing

Comment (tzneal) [reply pkg/controllers/provisioning/binpacking/packer.go]: Yup, good catch, bad copy/paste. Fixed.

**Comment by tzneal** [comment]
> 

Thanks! I was just reading the docs/code for familiarization and stumbled across it.

**Review by suket22 — APPROVED** [review]

**Pull request merged**